From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Mon, 8 Jun 2020 15:05:18 -0500
Subject: [PATCH] Linux 4.19.127-rt54 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade..3165a8781 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt54
-- 
https://clearlinux.org

