From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Tue, 28 Apr 2020 11:01:54 -0500
Subject: [PATCH] Linux 4.19.115-rt50 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade..42c384668 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt50
-- 
https://clearlinux.org

